src/renderer/src/routes/pages/Settings/index.tsx

```diff
@@ -0,0 +1,141 @@
+              }`}
+              onClick={() => setSelectedSidebarOption(SettingsSidebarOption.PERSONALISATION)}
+            >
+              {t('sidebarOption.personalisation')}
+            </button>
+            <button
+              className={`w-[50%] border-b ${
+                selectedSidebarOption === SettingsSidebarOption.SECURITY ? 'border-b-2' : 'text-gray-300'
+              }`}
+              onClick={() => setSelectedSidebarOption(SettingsSidebarOption.SECURITY)}
+            >
+              {t('sidebarOption.security')}
+            </button>
+          </div>
+          <nav className="flex flex-row justify-between h-15 w-full mb-5 text-[14px]">
+            <ul className="max-w-full w-full">
+              {selectedSidebarOption === SettingsSidebarOption.PERSONALISATION && (
+                <div>
+                  <SettingsSidebarLink
+                    id={SettingsPersonalisationOption.NETWORK_CONFIGURATION}
+                    title={t('personalisationOption.networkConfiguration')}
+                    icon={<Tb3DCubeSphere />}
+                    onClick={() => handlePersonalisationClick(SettingsPersonalisationOption.NETWORK_CONFIGURATION)}
+                    selectedId={selectedPersonalisationOption}
+                  />
+                  <SettingsSidebarLink
+                    id={SettingsPersonalisationOption.CURRENCY}
+                    title={t('personalisationOption.currency')}
+                    icon={<BsCash />}
+                    onClick={() => handlePersonalisationClick(SettingsPersonalisationOption.CURRENCY)}
+                    selectedId={selectedPersonalisationOption}
+                  />
+                  <SettingsSidebarLink
+                    id={SettingsPersonalisationOption.LANGUAGE}
+                    title={t('personalisationOption.language')}
+                    icon={<MdOutlineLanguage />}
+                    onClick={() => handlePersonalisationClick(SettingsPersonalisationOption.LANGUAGE)}
+                    selectedId={selectedPersonalisationOption}
+                  />
+                  <SettingsSidebarLink
+                    id={SettingsPersonalisationOption.THEME}
+                    title={t('personalisationOption.theme')}
+                    icon={<RiShapesLine />}
+                    onClick={() => handlePersonalisationClick(SettingsPersonalisationOption.THEME)}
+                    selectedId={selectedPersonalisationOption}
+                  />
+                  <SettingsSidebarLink
+                    id={SettingsPersonalisationOption.RELEASE_NOTES}
+                    title={t('personalisationOption.releaseNotes')}
+                    icon={<MdOutlineListAlt />}
+                    onClick={() => handlePersonalisationClick(SettingsPersonalisationOption.RELEASE_NOTES)}
+                    selectedId={selectedPersonalisationOption}
+                  />
+                </div>
```

Comment:
As you have the enum, you can use a loop here so you don't have to create one by one.

Outside the component create a new const for describe the title and icon for each option:

```
const infosByOption: Record<SettingsPersonalisationOption | SettingsSecurityOption, {title: string, icon: JSX.Element}> = {
   ENCRYPT_KEY: {
    title: 'Encrypt Key',
    icon: <MdOutlineLock />,
   },
...others
}
```

Now you can use a loop to render each option
```
             {
             selectedSidebarOption === SettingsSidebarOption.PERSONALISATION && Object.keys().map(option => {
             const info = infosByPersonalisationOption[option as SettingsPersonalisationOption]

             return (<SettingsSidebarLink
                    key={option}
                    title={info.title}
                    icon={info.icon}
                    onClick={() => handlePersonalisationClick(option)}
                    active={selectedSecurityOption === option}
                  />)
             })
             }
```